deploy/olm-catalog/opendatahub/0.9.0/opendatahub-operator.v0.9.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,151 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+
```

Comment:
Is the example empty on purpose?

Reply:
No. I updated it to include the [kfdef/kfctl_openshift.yaml](https://github.com/opendatahub-io/odh-manifests/blob/master/kfdef/kfctl_openshift.yaml)